site/src/components/NavbarView/NavbarView.tsx

```diff
@@ -19,41 +23,66 @@ export const Language = {
   users: "Users",
 }
 
-export const NavbarView: React.FC<NavbarViewProps> = ({ user, onSignOut }) => {
+const NavItems: React.FC<{ className?: string; linkClassName?: string }> = ({ className }) => {
   const styles = useStyles()
   const location = useLocation()
+
+  return (
+    <List className={combineClasses([styles.navItems, className])}>
```

Comment:
What's the benefit of using `combineClasses` instead of template literals?

Reply:
It handles, `null`, `undefined` and conditional styles like `{ isActive: styles.active }`.